tests/foreman/ui/test_host.py

```diff
@@ -2404,3 +2405,162 @@ def test_all_hosts_bulk_delete(target_sat, function_org, function_location, new_
         session.organization.select(function_org.name)
         session.location.select(function_location.name)
         assert session.all_hosts.bulk_delete_all()
+
+
+@pytest.fixture(scope='module')
+def change_content_source_prep(
+    module_target_sat,
+    module_sca_manifest_org,
+    module_capsule_configured,
+    module_location,
+):
+    """
+    This fixture sets up all the necessary entities for tests
+    exercising the Change of the hosts's content source.
+
+     It creates a new product in the organization,
+     creates a new repository in the product,
+     enables the specified repositories,
+     creates a new lce,
+     creates a new CV in the organization, adds the repository to the CV,
+     publishes the CV, and promotes the published version to the lifecycle environment,
+     creates a new activation key for the CV in the lce,
+     registers the RHEL content host with the activation key,
+     updates the capsule's taxonomies
+     adds the lifecycle environment to the capsule's content.
+
+     Fixture returns module_target_sat, org, lce, capsule, content_view, loc, ak
+    """
+    product_name, lce_name = (gen_string('alpha') for _ in range(2))
+    repos_to_enable = ['rhae2.9_el8']
```

Comment:
I wonder if this would show up as available on all parametrized rhel versions. Maybe using some universal custom repo or enabling one for each el version would enable tests using this fixture to check also content update from the host